ttnn/cpp/ttnn/operations/data_movement/clone/device/kernels/read_kernel_rm.cpp

```diff
@@ -2,35 +2,35 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#include <stdint.h>
-
 #include "dataflow_api.h"
 
 void kernel_main() {
-    uint32_t src_addr = get_arg_val<uint32_t>(0);
     uint32_t stick_size = get_arg_val<uint32_t>(1);
     uint32_t num_sticks = get_arg_val<uint32_t>(2);
     uint32_t start_id = get_arg_val<uint32_t>(3);
 
-    constexpr uint32_t cb_id_in0 = get_compile_time_arg_val(0);
-    constexpr bool src0_is_dram = get_compile_time_arg_val(1) == 1;
+    constexpr uint32_t src_cb_id = get_compile_time_arg_val(0);
+    constexpr bool input_is_dram = get_compile_time_arg_val(1) == 1;
 
-#define src_stick_size_is_pow2 get_compile_time_arg_val(2) == 1
-#if (src_stick_size_is_pow2)
+#define src_stick_size_is_power_of_two get_compile_time_arg_val(2) == 1
+#if (src_stick_size_is_power_of_two)
     constexpr uint32_t src_log_base_2_of_page_size = get_compile_time_arg_val(3);
-    const InterleavedPow2AddrGen<src0_is_dram> s0 = {
-        .bank_base_address = src_addr, .log_base_2_of_page_size = src_log_base_2_of_page_size};
+    const InterleavedPow2AddrGen<input_is_dram> s = {
+        .bank_base_address = get_arg_val<uint32_t>(0),
+        .log_base_2_of_page_size = src_log_base_2_of_page_size,
+    };
 #else
-    const InterleavedAddrGen<src0_is_dram> s0 = {.bank_base_address = src_addr, .page_size = stick_size};
+    const InterleavedAddrGen<input_is_dram> s = {
+        .bank_base_address = get_arg_val<uint32_t>(0),
+        .page_size = stick_size,
+    };
 #endif
 
     uint32_t end_id = start_id + num_sticks;
     for (uint32_t i = start_id; i < end_id; ++i) {
-        cb_reserve_back(cb_id_in0, 1);
-        uint32_t l1_write_addr = get_write_ptr(cb_id_in0);
-        uint64_t src_noc_addr = get_noc_addr(i, s0);
-        noc_async_read(src_noc_addr, l1_write_addr, stick_size);
+        cb_reserve_back(src_cb_id, 1);
+        noc_async_read(get_noc_addr(i, s), get_write_ptr(src_cb_id), stick_size);
```

Comment:
It doesn't seem like a problem, but directly passing `get_noc_addr(i, s)` into `noc_async_read` isn't good for debugging. 
If an issue arises, you'd need to revert to the previous style and create a variable for DPRINT, so it's better to avoid this approach.

Reply:
Seem reasonable, I agree, will reverse now.

Reply:
Changed.